python/requirements_compiled.txt

```diff
@@ -1844,7 +1844,7 @@ querystring-parser==1.2.4
     #   raydp
     #   tune-sklearn
     #   xgboost-ray
-raydp==1.7.0b20231020.dev0
+raydp==1.7.0b2022.9.0
```

Comment:
I'm not sure how the command `grep -l -R "3.0.0.dev0" * | grep -v "yaml" | grep -v "rst" | xargs -n 1 gsed -i 's/3.0.0.dev0/2.9.0/g'` causes this change, but it does.  I will revert this change.  Any thoughts on how to fix the command?

Reply:
this should be reverted.

the `grep` thing is terribly broke imo...